Guide_for_Contributors_New_to_Github.md

```diff
@@ -0,0 +1,124 @@
+
+Hi there! This guide aims to teach people unfamiliar with Github how to suggest edits to the [Tanzu Agile Practices site](https://tanzu.vmware.com/developer/practices/). Follow the steps below to get started!
+
+
+# Workflow overview
+
+It takes many clicks to suggest an edit, but these are the 5 big stages you have to go through.
+
+1. **Prepare**: Type up your suggested edits 
+1. **Fork**: Forking just means creating a copy of the practices site code for your GitHub account
+1. **Edit**: Copy and paste suggested edits into the code in your copy of the practices site through the GitHub UI and save it in a branch
+1. **Submit a Pull Request**: Submitting a Pull Request is the technical term for submitting the suggested edits to be reviewed, approved by someone else and updated on the site. It is so named because the reviewer will be 'pulling' your code into the vmware repository to merge it. 
+1. **Delete**: When the change is approved, delete your copy of the practices site.
+
+Repeat steps when you want to suggest new edits to the site.
+
+# Step by Step Instructions 
+
+## 1. Prepare
+1. Type up the changes you want to make for the Practices site. Any word editor will do for this, Microsoft Word, Google Docs, Notes on Mac. For your first suggestion, try adding something small, such as a few sentences. This will make it easier for the suggestion to be approved and merged.
```

Comment:
A few thoughts:
- maybe we should say "text editor" instead of "word editor"
- are we providing instructions on how to get the original content out of the git repo and into said editor?
- there are markdown-specific editors that render a nice live preview, which might be useful for someone not familiar with the syntax - maybe we should recommend one 
- instead of saying "try adding something small" can we say something along the lines of "We encourage small changes , which are easier to review and merge. If you have a large amount of edits to make, consider breaking them up into a series of smaller changes."